src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -64,6 +65,7 @@ public void Add(Cookie cookie)
             {
                 throw new ArgumentNullException(nameof(cookie));
             }
+            ++m_version;
```

Comment:
I was thinking more about a Windows-specific feature addition (b) like a new public method which a) wouldn't break compatibility with previous netfx versions (a). Because of (b) it wouldn't be ported to Core.